pkg/pulumiyaml/run_options_test.go

```diff
@@ -64,9 +64,13 @@ configuration:
   shouldProtect:
     default: false
     type: boolean
+  version:
+    default: "1.0.0"
+    type: string
 resources:
   provider-a:
     type: pulumi:providers:test
+    version: ${version}
   provider-b:
```

Comment:
Shouldn't something later in this test assert that provider-a is actually registered with version "1.0.0"?

Reply:
Added an assertion that the registered provider has the version evaluated correctly ✅